cf_remote/commands.py

```diff
@@ -931,3 +932,16 @@ def agent(hosts, bootstrap=None):
             print(output)
 
     return 0
+
+
+def connect_cmd(hosts):
+
+    assert hosts and len(hosts)
```

Comment:
@victormlg This is highly subjective, of course, but to me, my suggestion was a bit more clear;
```suggestion
    assert hosts and len(hosts) >= 1 # Ensured by argument parser
```

The behavior is exactly the same, but I don't have to think so much about what `len(hosts)` can be, and it also has the comment to show what this assumption is based on.